src/grpc/rpc_server.cpp

```diff
@@ -31,13 +31,18 @@ SISL_LOGGING_DEF(grpc_server)
 namespace sisl {
 GrpcServer::GrpcServer(const std::string& listen_addr, uint32_t threads, const std::string& ssl_key,
                        const std::string& ssl_cert) :
-        GrpcServer::GrpcServer(listen_addr, threads, ssl_key, ssl_cert, nullptr) {}
-
+        GrpcServer::GrpcServer(listen_addr, threads, 0, ssl_key, ssl_cert, nullptr) {}
 GrpcServer::GrpcServer(const std::string& listen_addr, uint32_t threads, const std::string& ssl_key,
                        const std::string& ssl_cert, const std::shared_ptr< sisl::GrpcTokenVerifier >& auth_mgr) :
+        GrpcServer::GrpcServer(listen_addr, threads, 0, ssl_key, ssl_cert, auth_mgr) {}
+GrpcServer::GrpcServer(const std::string& listen_addr, uint32_t threads, int max_receive_msg_size,
+                       const std::string& ssl_key, const std::string& ssl_cert,
+                       const std::shared_ptr< sisl::GrpcTokenVerifier >& auth_mgr) :
         m_num_threads{threads}, m_auth_mgr{auth_mgr} {
     if (listen_addr.empty() || threads == 0) { throw std::invalid_argument("Invalid parameter to start grpc server"); }
 
+    if (max_receive_msg_size != 0) { m_builder.SetMaxReceiveMessageSize(max_receive_msg_size); }
+
```

Comment:
I suggest to improve the echo_async_client.cpp test to include this option as well. My suggestion would be to add this option to in here
https://github.com/eBay/sisl/blob/master/src/grpc/tests/function/echo_async_client.cpp#L420
```
m_grpc_server = GrpcServer::make(server_address, 4, <64MB>, "", "");
````

and update the 
```
static const std::string GENERIC_CLIENT_MESSAGE{"I am a super client!"};
```
from a simple message to a message of 63.9MB size and ensure we are able to successfully send.

Reply:
If you are looking for random string, there could be many ways, we have one here:
https://github.com/eBay/sisl/blob/c3b8844f7be32fddfc3baca986f6508e74d3211d/src/grpc/tests/unit/client_test.cpp#L42

Reply:
ack will do

Reply:
done @hkadayam